app/controllers/campaigns_controller.rb

```diff
@@ -83,6 +89,10 @@ def add_organizer_to_campaign
                           role: CampaignsUsers::Roles::ORGANIZER_ROLE)
   end
 
+  def is_organizer?
+    current_user.is_a?(CampaignsUsers) && current_user.role == CampaignsUsers::Roles::ORGANIZER_ROLE
```

Comment:
`current_user` is a `User` object. Maybe instead set @campaigns_user first and then check the role on that.